### PR TITLE
{tools}[dummy/] vsc-mympirun v4.0.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2412.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/36178e1b6f6d739a349347567f9fb434 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2005.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cd573fed40ab2342b687091321ac6f7d for a full test report.

File: easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.2.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# we ship something in bin/fake
+modextrapaths = {'PATH': 'bin/fake'}
+
+# don't check for import of vsc.mympirun, since that may fail if vsc-base is picked up from the OS
+options = {'modulename': 'vsc'}
+
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[:2])
+sanity_check_paths = {
+    'files': ['bin/mympirun', 'bin/mympisanity'],
+    'dirs': ['bin/fake', 'lib/python%s/site-packages' % pyshortver],
```
Comment (wpoely86) [inline easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.2.eb]: don't we have constants for this?

Comment (boegel) [reply easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.2.eb]: `%(pyver)s` & co only work when Python is included as a (proper) dependency, not when it's a system dep

**Pull request merged**